modules/node-function-app/test/variables.tf

```diff
@@ -19,17 +19,17 @@ variable "app_service_plan_id" {
   type        = string
 }
 

```

Comment:
again, i can delete these. Not sure why TF Validate was not picking all this up before

Reply:
Not a massive concern as in the test folder